docs/docs/userguide/MIGRATION.md

```diff
@@ -2,7 +2,7 @@
 
 If you already have an existing Data Center product deployment, you can migrate it to a Kubernetes cluster using the Data Center Helm charts. 
 
-You will need to migrate your database and your shared home, then all you need to do is to follow the [Installation guide](INSTALLATION.md), using your migrated resources instead of provisioning new ones.
+You will need to migrate your database and your shared home (including local home for Bamboo), then all you need to do is to follow the [Installation guide](INSTALLATION.md), using your migrated resources instead of provisioning new ones.
```

Comment:
Why we should mention local home in parenthesis? I think we can say `database, shared home, and local home`

Reply:
/nit - can we link `[Installation guide](INSTALLATION.md)` to the import heading?

Reply:
> /nit - can we link [Installation guide](INSTALLATION.md) to the import heading?

@nanux I'm not following you on this?

Reply:
@nghazali `local-home` does not need to be migrated by the other products, this is specific to Bamboo hence the parentheses.